lib/Sheet.js

```diff
@@ -1,5 +1,5 @@
b-active",t:13,f:[{t:2,x:{r:["~/selected"],s:"_0===\"output\""}}]}],f:["Output"]}," ",{t:7,e:"li",m:[{n:"class",f:"rs-tab",t:13},{n:["click"],t:70,f:{r:[],s:"[[\"select\",\"console\"]]"}},{n:"class-rs-tab-active",t:13,f:[{t:2,x:{r:["~/selected"],s:"_0===\"console\""}}]}],f:["Console"]}," ",{t:4,f:[{t:7,e:"li",m:[{n:"class",f:"rs-button",t:13},{n:["click"],t:70,f:{r:["@this","popped"],s:"[_0.set(\"popped\",_1===2?1:0)]"}}],f:["▼"]}," ",{t:7,e:"li",m:[{n:"class",f:"rs-button",t:13},{n:["click"],t:70,f:{r:["@this"],s:"[_0.set(\"popped\",2)]"}},{n:"class-rs-disabled",t:13,f:[{t:2,x:{r:["popped"],s:"_0===2"}}]}],f:["▲"]}," ",{t:4,f:[{t:7,e:"li",m:[{n:"class",f:"rs-button",t:13},{n:["click"],t:70,f:{r:["@this"],s:"[_0.set(\"messages\",[])]"}}],f:["∅"]}],n:50,x:{r:["selected"],s:"_0===\"console\""}}],n:50,r:"popped"}]}," ",{t:7,e:"ul",m:[{n:"class",f:"rs-contents",t:13}],f:[{t:7,e:"li",m:[{n:"class",f:"rs-content rs-output",t:13},{n:"class-rs-content-active",t:13,f:[{t:2,x:{r:["~/selected"],s:"_0===\"output\""}}]}],f:[{t:7,e:"iframe",m:[{n:"name",f:"playground output",t:13},{n:"sandbox",f:"allow-scripts allow-forms allow-same-origin allow-modals",t:13}]}]}," ",{t:7,e:"li",m:[{n:"class",f:"rs-content rs-console",t:13},{n:"class-rs-content-active",t:13,f:[{t:2,x:{r:["~/selected"],s:"_0===\"console\""}}]}],f:[{t:7,e:"ul",f:[{t:4,f:[{t:4,f:[{t:19,f:[{t:7,e:"li",m:[{n:"class",f:["rs-message ",{t:2,x:{r:[".type"],s:"_0?(\"rs-message-\"+(_0)):\"\""}}],t:13}],f:[{t:7,e:"pre",f:[{t:7,e:"code",f:[{t:4,f:[{t:2,r:".message"}],n:50,r:"__state.show"},{t:4,n:51,f:[{t:2,x:{r:[".message"],s:"_0.substring(0,_0.indexOf(\"\\n\"))"}}],l:1}]}]}," ",{t:7,e:"button",m:[{n:["click"],t:70,f:{r:["@context"],s:"[(_0).toggle(\"__state.show\")]"}}],f:["..."]}]}],n:54,z:[{n:"__state",x:{r:"@local"}}]}],n:50,x:{r:["message"],s:"~_0.indexOf(\"\\n\")"}},{t:4,n:51,f:[{t:7,e:"li",m:[{n:"class",f:["rs-message ",{t:2,x:{r:[".type"],s:"_0?(\"rs-message-\"+(_0)):\"\""}}],t:13}],f:[{t:7,e:"pre",f:[{t:7,e:"code",f:[{t:2,r:".message"}]}]}]}],l:1}],n:52,z:[{n:"message",x:{r:"."}}],r:"messages"}]}]}]}]}],e:{"!!_0":function (_0){return(!!_0);},"[[\"select\",\"output\"]]":function (){return([["select","output"]]);},"_0===\"output\"":function (_0){return(_0==="output");},"[[\"select\",\"console\"]]":function (){return([["select","console"]]);},"_0===\"console\"":function (_0){return(_0==="console");},"[_0.set(\"popped\",_1===2?1:0)]":function (_0,_1){return([_0.set("popped",_1===2?1:0)]);},"[_0.set(\"popped\",2)]":function (_0){return([_0.set("popped",2)]);},"_0===2":function (_0){return(_0===2);},"[_0.set(\"messages\",[])]":function (_0){return([_0.set("messages",[])]);},"_0?(\"rs-message-\"+(_0)):\"\"":function (_0){return(_0?("rs-message-"+(_0)):"");},"_0.substring(0,_0.indexOf(\"\\n\"))":function (_0){return(_0.substring(0,_0.indexOf("\n")));},"[(_0).toggle(\"__state.show\")]":function (_0){return([(_0).toggle("__state.show")]);},"~_0.indexOf(\"\\n\")":function (_0){return(~_0.indexOf("\n"));}}},
```

Comment:
It looks like the non-ascii chars came out unescaped, which will blow up mkdocs. You can pass `-u` to the ractive bin or `escapeUnicode: true` to the lib to spit them out escaped.